src/components/Home/index.jsx

```diff
@@ -333,32 +615,103 @@ export default class Home extends React.Component {
               <Car src={car} alt="car" position={SCROLL_POS} />
               <ForegroundBush src={foregroundBush} alt="foregroundBush" />
             </RoadWrapper>
-            <TimeWrapper>
-              {CLICKABLES.map(e => (
-                <Clickable
-                  key={e.title}
-                  onClick={() => this.changeFAQ(e.title)}
-                >
-                  {FAQ_STATE
-                    === e.title ? <b>{e.title}</b> : e.title}
-                </Clickable>
-              ))}
-            </TimeWrapper>
-            <FAQContainer>
-              {FAQ_PANELS[FAQ_STATE
-              ].content.map(e => (
-                <FAQTitle key={e[0].q}>
-                  {e.map(f => (
-                    <div key={f.q}>
-                      <b>{f.q}</b><br />{f.a}<br /><br />
-                    </div>
+            <FAQMobileArrows
+              invisible={FAQ_STATE === CLICKABLES[0].title}
+              isLeft
+              onClick={this.scrollLeft}
+            >&#8249;
+            </FAQMobileArrows>
+            <FAQMobileArrows
+              invisible={FAQ_STATE === CLICKABLES[CLICKABLES.length - 1].title}
+              onClick={this.scrollRight}
+            >&#8250;
+            </FAQMobileArrows>
+            <FAQHeightMaintainer>
+              <FAQMaxHeight>
+                <FAQPlaceholder>
+                  {FAQ_PANELS.General.content.map(e => (
+                    <FAQTitle key={e[0].q}>
+                      {e.map(f => (
+                        <div key={f.q}>
+                          <b>{f.q}</b><br />{f.a}<br /><br />
+                        </div>
+                      ))}
+                    </FAQTitle>
+                  ))}
+                </FAQPlaceholder>
+                <FAQPlaceholder>
+                  {FAQ_PANELS.Before.content.map(e => (
+                    <FAQTitle key={e[0].q}>
+                      {e.map(f => (
+                        <div key={f.q}>
+                          <b>{f.q}</b><br />{f.a}<br /><br />
+                        </div>
+                      ))}
+                    </FAQTitle>
+                  ))}
+                </FAQPlaceholder>
```

Comment:
Again, could probably be simplified by using an array:
```
CLICKABLES.map(section => (
  <FAQPlaceholder>
    {FAQ_PANELS[section.title].content.map(e => (
      ...
    ))}
  </FAQPlaceholder>
))
```

Also, maybe use better variable names instead of `e` and `f` 😄